### DISCUSSION TITLE
OAuthCallbackError - Twitter Provider

### DISCUSSION THREAD

**Discussion by ephraimduncan** [nextauthjs/next-auth#4550] [Help]
### Provider type

Twitter

### Environment

_System_
    OS: Linux 5.15 Manjaro Linux
    CPU: (2) x64 Intel(R) Celeron(R) CPU  N3060  @ 1.60GHz
    Memory: 858.74 MB / 3.74 GB
    Container: Yes
    Shell: 5.8 - /bin/zsh

  _Binaries_
    Node: 16.13.1 - /run/user/1000/fnm_multishells/4779_1651959095521/bin/node
    Yarn: 1.22.17 - /run/user/1000/fnm_multishells/4779_1651959095521/bin/yarn
    npm: 8.1.2 - /run/user/1000/fnm_multishells/4779_1651959095521/bin/npm
 
_Browsers_
    Firefox: 97.0.1

_npmPackages_
    next: ^12.0.11-canary.4 => 12.1.6 
    next-auth: latest => 4.3.4 
    react: ^17.0.2 => 17.0.2 

### Reproduction URL

https://github.com/nextauthjs/next-auth-example

### Describe the issue

I had this issue on a project. I decided to clone the next-auth-example. I am still getting the same problem. A user logs in using Twitter as OAuth. I keep getting OAuthCallBackError.

```
[next-auth][error][CALLBACK_OAUTH_ERROR] 
```

### How to reproduce

I have no idea to reproduce

### Expected behavior

It should redirect back to the application when I sign-in.

**Comment by coding-velociraptor** [comment]
I just wanted to confirm that I'm encountering the same issue. I do not have steps to reproduce so far. It does not happen with all users, but only a subset. Consistently across browsers and after removing the Twitter OAuth App from their account.
